release_bot/github.py

```diff
@@ -298,6 +299,11 @@ def make_pr(self, branch, version, log, changed_version_files, base='master'):
         if response.status_code == 201:
             parsed = response.json()
             self.logger.info(f"Created PR: {parsed['html_url']}")
+
+            # put labels on PR
```

Comment:
Does it work on PR too ?

Reply:
Yes it does. According to Github API docs:
https://developer.github.com/v3/pulls/#labels-assignees-and-milestones

> Every pull request is an issue, but not every issue is a pull request. For this reason, "shared" 
> actions for both features, like manipulating assignees, labels and milestones, are provided 
> within the [Issues API](https://developer.github.com/v3/issues).